### PR TITLE
Scholarship dropdowns set course

### PR DESCRIPTION
**Pull request by clareconstantine**
Part of [PLC-165](https://codedotorg.atlassian.net/browse/PLC-165)

Now that we have added the `course` column to `ScholarshipInfo`, the existing scholarship dropdowns need to set it.

### CODE REVIEW

File: dashboard/app/models/pd/scholarship_info.rb [hunk 1]
```diff
@@ -39,8 +39,8 @@ class Pd::ScholarshipInfo < ActiveRecord::Base
   validates_inclusion_of :application_year, in: SCHOLARSHIP_YEARS
   validates_inclusion_of :scholarship_status, in: SCHOLARSHIP_STATUSES
 
-  def self.update_or_create(user, application_year, scholarship_status)
-    scholarship_info = Pd::ScholarshipInfo.find_by(user: user, application_year: application_year) || Pd::ScholarshipInfo.new(user: user)
-    scholarship_info.update(scholarship_status: scholarship_status)
+  def self.update_or_create(user, application_year, course, scholarship_status)
```
Comment (breville) [inline dashboard/app/models/pd/scholarship_info.rb:42]: At a certain number of parameters, a hash might be better?  Not sure we're there yet, though...

File: dashboard/app/models/pd/application/teacher_application_base.rb [hunk 2]
```diff
@@ -65,11 +65,13 @@ def update_user_school_info!
     end
 
     def update_scholarship_status(scholarship_status)
-      Pd::ScholarshipInfo.update_or_create(user, application_year, scholarship_status)
+      Pd::ScholarshipInfo.update_or_create(user, application_year, course_name, scholarship_status)
```
Comment (breville) [inline dashboard/app/models/pd/application/teacher_application_base.rb]: Is `course_name` a user-friendly string (e.g. `"CS Discoveries"`) or a short code (e.g. `"csd"`)?

Comment (clareconstantine) [reply dashboard/app/models/pd/application/teacher_application_base.rb]: user-friendly, so it matches the `course` we store for workshops.

Comment (clareconstantine) [reply dashboard/app/models/pd/application/teacher_application_base.rb]: but based on your feedback [here](https://github.com/code-dot-org/code-dot-org/pull/28284#issuecomment-488107680) I'm going to switch to the simplfied course code ('csd')

**Review by hacodeorg — APPROVED** [review]

**Pull request merged**